Source/Launcher/Program.cs

```diff
@@ -1,13 +1,5 @@
 ﻿using System;
```

Comment:
Why do we still have a launcher in the core project on its own? Since there is now the convenience API in the core and the launcher code in the template, this should be obsolete?

Reply:
We still need it to debug during developing duality ourselves. However we could just reference the same source files.

Reply:
The same source code files as in the template are now being used